roofit/roofitcore/src/Initialisation.cxx

```diff
@@ -38,9 +38,16 @@ void doBanner() {
 ////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 /// Inspect cpu capabilities, and load architecture-specific libraries for RooFitCore/RooFit computations.
 void loadComputeLibrary() {
+  if (gEnv->GetValue("RooFit.LoadOptimisedComputationLibrary", 1) == 0) {
+    if (gDebug>0) std::cout << "RooFit.LoadOptimisedComputationLibrary is set to 0, so skipping loading any library" << std::endl;
+    return;
+  }
+#ifdef R__RF_ARCHITECTURE_SPECIFIC_LIBS
+
   std::vector<std::string> libNames;
+
 //  libNames.push_back("libRooFitCoreCompute");
```

Comment:
Is that needed? This whole section seems to consist of a private branch pushed into master as commented code ;-) Can you keep the master +/- clean, and keep these kind of "that's what I want to do in the future" changes in a private branch, possibly on your fork of ROOT?

Esp the part starting with `// Try to load HistFactory compute library as well.` below could be removed.